modules/genomic_browser/php/snp_browser.class.inc

```diff
@@ -148,15 +148,16 @@ class SNP_Browser extends \NDB_Menu_Filter
         if (!$user->hasPermission('genomic_browser_view_allsites')) {
             // restrict data to own site
             $site_arr     = implode(",", $user->getCenterIDs());
-            $this->query .= " AND candidate.CenterID IN (" . $site_arr . ")";
+            $this->query .= " AND candidate.RegistrationCenterID IN ("
```

Comment:
should also maybe be the session centerID

Reply:
Same comment. I don't think there is any sessionID to attach genomic data. @xlecours could you confirm?